acceptance/redis32/redis-ha.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+
+set -xue
+
+
+# get all pods by service_id
+curl -ks -u"${K8S_USERNAME}:${K8S_PASSWORD}" "${K8S_APISERVER}/api/v1/namespaces/default/pods/" | \
+jq '.items[] | select( .metadata.name | test( "'"${SERVICE_ID}"'" ) ) | { name: .metadata.name, node: .status.hostIP, ip: .status.podIP }'
```

Comment:
Could we do this filtering in the api call instead of in post-processing using label selectors?